src/ftest.jl

```diff
@@ -1,3 +1,10 @@
+struct SingleFTestResult{T<:Real}
+    df1::T
+    df2::T
+    f::T
+    p::T
```

Comment:
Like for `FTest`, we can require that all values are `Float64`, as it's not useful to use other types in practice. Then the `T` type parameter can be dropped completely.
```suggestion
    f::Float64
    p::Float64
```